pages/Advanced Queries.md

```diff
@@ -138,6 +138,33 @@ description:: Advanced queries are written with [Datalog](https://en.wikipedia.o
 			- `:Xd-after-ms` - Timestamp of X days after today. **New syntax** `:+Xd-ms`
 			- `:start-of-today-ms` - Timestamp for beginning of today. **New syntax** `:today-start`
 			- `:end-of-today-ms` - Timestamp for end of today. **New syntax** `:today-end`
+	- ### Query parameters and functions
```

Comment:
Most of this section is already written in https://docs.logseq.com/#/page/queries and putting it in this page is unfortunately incorrect. There are already a couple of references to "simple query" in this page. I could link simple query to https://docs.logseq.com/#/page/queries if that helps